charts/retool/templates/deployment_code_executor.yaml

```diff
@@ -53,7 +53,11 @@ spec:
         image: "{{ .Values.codeExecutor.image.repository }}:{{ include "retool.codeExecutor.image.tag" . }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         securityContext:
+          {{ if .Values.codeExecutor.securityContext }}
+          {{ toYaml .Values.codeExecutor.securityContext }}
+          {{ else }}
           privileged: true
+          {{ end }}
```

Comment:
we can drop this else block if the values.yaml file is always included as part of using new helm version (not sure about this)